README.Rmd

```diff
@@ -77,3 +77,27 @@ Please note that although encouraged, it is not strictly mandatory to submit the
 code generating your forecasts. You can submit a script fetching already
 existing forecasts from another location such as an API, another GitHub
 repository, a data repository, a dropbox folder, etc.
+
+
+---
+
+### Developer guidelines
+
+Update submodules with
+
+```
+git submodule foreach git pull
+
+```
+
+Add a new submodule with
+
+```
+git submodule add <repo_url> models/repo_name
+```
+
+Remove a submodule with
+
+```
+git submodule deinit models/repo_name
+git rm models/repo_name
```

Comment:
This should potentially be part of the wiki, which aims at having all the instructions for maintainers.